DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/Venditore/GestioneOrdini.tex

```diff
@@ -1,12 +1,201 @@
+\begin{itemize}
+	\item \textbf{Attori primari:} venditore;
+	\item \textbf{Precondizione:} il venditore è nella schermata di riepilogo ordini ed ha intenzione di cambiare ordine con cui visualizzare gli ordini;
+	\item \textbf{Postcondizione:} il venditore visualizzerà nella schermata di riepilogo ordini gli ordini a suo carico in base al metodo da lui selezionato;
+	\item \textbf{Scenario principale:} il venditore vuole visualizzare gli ordini cronologicamente ed esegue l'operazione che li riordina dal più al meno recente o viceversa;
+	\item \textbf{Scenari alternativi:}
+	\begin{enumerate}[label=\lett]
+		\item Se il venditore non esegue l'operazione di riordinamento, allora la schermata di riepilogo ordini non verrà aggiornata.
+	\end{enumerate}
+\end{itemize}
+
+\subUC{Modifica stato ordine}
+Il venditore vuole modificare lo stato di un determinato ordine.
+\begin{itemize}
+	\item \textbf{Attori primari:} venditore;
+	\item \textbf{Precondizione:} il venditore ha intenzione di modificare lo stato di un ordine a suo carico;
+	\item \textbf{Postcondizione:} il venditore ha modificato lo stato di un ordine;
+	\item \textbf{Scenario principale:}
+	\begin{itemize}
+		\item Il venditore seleziona un ordine dalla lista;
+		\item Il venditore modifica lo stato dell'ordine.
+	\end{itemize}
+\end{itemize}
+
+\subUC{Visualizzazione dettagli ordine}
+Il venditore vuole visualizzare i dettagli di un determinato ordine.
+\begin{itemize}
+	\item \textbf{Attori primari:} venditore;
+	\item \textbf{Precondizione:} il venditore vuole vedere i dettagli di un determinato ordine a suo carico;
+	\item \textbf{Postcondizione:} il venditore visualizza tutti i dettagli dell'ordine;
+	\item \textbf{Scenario principale:}
+	\begin{itemize}
+		\item Il venditore seleziona un ordine dalla lista;
+		\item Il venditore apre la schermata con i dettagli dell'ordine selezionato.
+	\end{itemize}
+\end{itemize}
+
+\UC{Ricerca di un ordine per codice} 
+Il venditore può cercare un ordine dalla propria schermata di riepilogo ordini.
+\begin{itemize}
+	\item \textbf{Attori primari:} venditore;
+	\item \textbf{Precondizione:} il venditore ha selezionato la funzionalità per la ricerca e l'inserimento del codice per individuare un ordine;
+	\item \textbf{Postcondizione:} il venditore visualizza l'ordine che corrisponde al codice per il quale si è svolta la ricerca;
+	\item \textbf{Scenario principale:} il venditore ha selezionato la funzione prevista per la ricerca. Dopo aver inserito il codice per individuare l'ordine, conferma la ricerca e viene aggiornata la schermata di riepilogo ordini che mostra solamente l'ordine corrispondente al codice indicato;
+	\item \textbf{Estensioni:}
+	\begin{enumerate}[label=\lett]
+		\item Il venditore ha svolto una ricerca che non ha prodotto alcun risultato. In questo caso:
+		\begin{itemize}
+			\item (UC) - Viene visualizzato il messaggio di ricerca senza alcun risultato.
```

Comment:
A documento completo ricordarsi di sistemare perché c'è un uc uguale per l'acquirente e scritto in modo diverso